projects/ADuCM360_demo_cn0411/system/adi_initialize.c

```diff
@@ -0,0 +1,22 @@
+/*
+** adi_initialize.c source file generated on October 18, 2017 at 18:13:52.
+**
+** Copyright (C) 2000-2017 Analog Devices Inc., All Rights Reserved.
```

Comment:
Copyright year should be 2018 now?

Reply:
There are several files with older copyrights. I updated only the files that I've modified. 
` The general rule is that the year to include in a copyright notice is the year of first publication of the work. `